### PR TITLE
Obstacle course fifth

### PR DESCRIPTION
**Pull request by isabellammjfk**
so basically the robot moves towards the wall, turns, goes forward, then goes forward to yeet 16177 out of the way to park, ending the obstacle course 

starting position: (-5, -30) 
ending position: (40,-35) 
Screenshot 2024-09-03 164348

### CODE REVIEW

**Comment by OLeg25685** [comment]
Gooood work! if do little cleaning in code it will look perfect.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package org.firstinspires.ftc.teamcode.drive.opmode;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.geometry.Vector2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
```
Comment (OLeg25685) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java:8]: Why there doing this space?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        waitForStart();
+
+        if(isStopRequested()) return;
+
+
+        drive.followTrajectory(forwardToWall);
+        drive.followTrajectory(forwardToMosaic);
+        drive.followTrajectory(yeetTheRobotToPark);
+    }
+}
```
Comment (OLeg25685) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java:41]: All else code is cool

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import org.firstinspires.ftc.teamcode.drive.SampleMecanumDrive;
+
+@Autonomous (group = "drive")
+public class izzyPath extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        SampleMecanumDrive drive = new SampleMecanumDrive(hardwareMap);
+
+
+        //
```
Comment (OLeg25685) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java]: What and why there those comment, where is comment?

**Review by OLeg25685 — CHANGES_REQUESTED** [review]
There is little trouble for my eyes

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java
Comment (CubeOfCheese) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java:1]: Your file should not be named after you. It should be named ObstacleCoursePath.java

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+@Autonomous (group = "drive")
+public class izzyPath extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        SampleMecanumDrive drive = new SampleMecanumDrive(hardwareMap);
+
+
+        drive.setPoseEstimate(new Pose2d(-5, -30, Math.toRadians(270)));
+
+        Trajectory forwardToWall= drive.trajectoryBuilder(new Pose2d(-5, -30, Math.toRadians(270)))
```
Comment (CubeOfCheese) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java:20]: space before equal sign plz
```suggestion
        Trajectory forwardToWall = drive.trajectoryBuilder(new Pose2d(-5, -30, Math.toRadians(270)))
```

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java (hunk 2 above)
Comment (AkithMadurapperuma) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java:41]: GOOD!

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java (hunk 1 above)
Comment (AkithMadurapperuma) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java:8]: GREAT!

**Review by CubeOfCheese — CHANGES_REQUESTED** [review]

**Pull request merged**